.github/workflows/database.yml

```diff
@@ -14,6 +14,39 @@ on:
     - cron:  '0 7 * * *'
 
 jobs:
+  unit:
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        target: [iOS, tvOS, macOS]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Bundler
+      run: scripts/setup_bundler.sh
+    - name: BuildAndTest # can be replaced with pod lib lint with CocoaPods 1.10
+      run: scripts/third_party/travis/retry.sh scripts/build.sh Database ${{ matrix.target }} unit
```

Comment:
Still doing the retry here - but yep - subsequent tries typically get worse - there seems to be something fundamental that needs to be fixed with these tests - but not clear to me after a bit of debugging ...

Reply:
After retrying for a couple hours last week I realized I should have spent that time fixing the flakes. But then the tests passed :)